src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -344,12 +344,12 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         enhancer.setSuperclass(delegate.getClass())
         enhancer.setCallback([
 
-                invoke: {Object proxy, Method method, Object[] args ->
-                    if ("onNext" == method.name) {
-                        logger.info(args[0].stream)
-                    }
-                    method.invoke(delegate, args)
+            invoke: {Object proxy, Method method, Object[] args ->
+                if ("onNext" == method.name) {
+                    logger.quiet(args[0].toString())
```

Comment:
@cdancy, I can't test right now if that was really the case, but I think `toString` returns all the information related to the ResultItem (in a format like "Person[name=John Doe,age=33,smoker=false]"). I opted for using just the `stream` field so it reflected more the way docker output looks. Can you check if that is the case?

Can't think of why you're getting the StreamCorruptedException as `getStream` does nothing more than return the field value. If you can show me how to reproduce it, I can have a look tomorrow.